fedora-ci/jobs.md

```diff
@@ -16,4 +16,5 @@ in [Testing Farm](https://testing-farm.io) run with a successfull scratch build.
 
 ### Custom test plans
 
-TBD (The work is planned.)
+Custom [TMT](https://tmt.readthedocs.io/) test plans in [Testing Farm](https://testing-farm.io)
+run with a successfull scratch build.
```

Comment:
Can we maybe mention what we mean by that? (I.e. should it be present in the repo? Do I, as a user, need to configure anything....

Reply:
@lachmanfrantisek WDYT?

Reply:
Thanks! Looks good.